eng/scripts/sdk_tests.sh

```diff
@@ -12,4 +12,6 @@ rustup update --no-self-update ${BUILD}
 export RUSTFLAGS="-Dwarnings"
 cargo +${BUILD} check -p azure_core --no-default-features
 cargo +${BUILD} check --all --features azurite_workaround
+cargo +${BUILD} check --all --all-targets --all-features
 cargo +${BUILD} test --all --features hmac_rust
+cargo +${BUILD} test --all --all-targets --all-features
```

Comment:
Do you know which targets generated the errors? I'm surprised we hadn't run into them before, and it may be better to test those targets in parallel rather than every time this script is run. Besides, there's some major changes coming soon and these scripts will likely be removed in favor of parallel jobs. I want to make sure we cover what failed.

Reply:
The target generating the error is `client_certificate` from `src/identity`. I was just using --all-features to check I wasn't breaking anything else when I was making changes. I can change change the script to just include that feature, and then you can run all features in parallel when the your major changes land.

Reply:
My repo https://dev.azure.com/msazuredev/AzureForOperators/_git/a40na-ingestion-file-uploader is also hitting this issue after updating 0.19.0 -> 0.20.0 because we use the `client_certificate` feature.

Reply:
> I can change change the script to just include that feature, and then you can run all features in parallel when the your major changes land.

@akinnane yes, can you make that change instead?